be/src/olap/rowset/segment_creator.cpp

```diff
@@ -245,17 +252,16 @@ Status SegmentFlusher::_flush_segment_writer(
     uint32_t segment_id = writer->segment_id();
     SegmentStatistics segstat;
     segstat.row_num = row_num;
-    segstat.data_size = segment_size + writer->get_inverted_index_total_size();
-    segstat.index_size = index_size + writer->get_inverted_index_total_size();
+    segstat.data_size = segment_size;
+    segstat.index_size = inverted_index_file_size;
```

Comment:
Is this change the origin meaning of index size?

Reply:
yes!